src/Legacy/Storage.php

```diff
@@ -1831,12 +1817,11 @@ protected function executeGetContentQueries($decoded)
      *
      * @param string $textquery
      * @param string $parameters
-     * @param array  $pager
      * @param array  $whereparameters
      *
      * @return array
      */
-    public function getContent($textquery, $parameters = '', &$pager = [], $whereparameters = [])
+    public function getContent($textquery, $parameters = '', $whereparameters = [])
     {
```

Comment:
Not sure that changing the signature on this is a good idea. This whole class is deprecated, but we're trying to make the transitional in 3.x and remove it all in 4.0